src/ui/FormRadio/index.tsx

```diff
@@ -0,0 +1,72 @@
+// import FORM_RULES from '@/constants/FormRule';
+import {
+  FormControl,
+  FormLabel,
+  // useTheme,
+  RadioGroup,
+  Radio,
+  Stack,
+  // HStack,
+  Flex,
+} from '@chakra-ui/react';
+import { useFormContext } from 'react-hook-form';
```

Comment:
```suggestion
import {
  FormControl,
  FormLabel,
  RadioGroup,
  Radio,
  Stack,
  Flex,
} from '@chakra-ui/react';
import { useFormContext } from 'react-hook-form';
```

p2;
사용하지 않는다면 아예 주석을 지워도 될 것 같아요